src/adservice/Dockerfile

```diff
@@ -16,6 +16,7 @@ FROM adoptopenjdk/openjdk11
 ENV OTEL_SERVICE_NAME=AdService
 ENV OTEL_LOGS_EXPORTER=otlp
 ENV OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE=DELTA
+ENV OTEL_EXPORTER_OTLP_METRICS_DEFAULT_HISTOGRAM_AGGREGATION=EXPONENTIAL_BUCKET_HISTOGRAM
```

Comment:
Does this mean the AdService will no longer report any explicit bucket histograms?

Reply:
Correct.

Reply:
Does it need to?  Or are there already other services reporting explicit bucket histograms?

Reply:
There are others reporting explicit bucket histograms (frontend and cartservicex).